### PR TITLE
Update UG, DG on NFR & UG Add Feature

### LINKED ISSUE #34: Update AboutUs, Skeletal PPP (Elaine)

### LINKED ISSUE #38: Update UG Add feature

### LINKED ISSUE #40: Update DG on NFR

### PR DESCRIPTION
**Pull request by elaineshijie**
Fixes #40 
Fixes #38 
Fixes #34

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -45,22 +45,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 * Responsibilities: Data
 
-### Jean Doe
+### Loh Shi Jie, Elaine
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/elaineshijie.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
```
Comment (pzl111) [inline docs/AboutUs.md:53]: Is there supposed to be a link to your PPP?

**Review by pzl111 — COMMENTED** [review]
Might have missing link to PPP?

File: docs/AboutUs.md (hunk 1 above)
Comment (elaineshijie) [reply docs/AboutUs.md:53]: oh opps I added in in the next commit

**Review by pzl111 — APPROVED** [review]
LGTM

**Pull request merged**